Parsers/ASimAuthentication/ProductParsers/AuthenticationWindowsEvents.yaml

```diff
@@ -0,0 +1,136 @@
+                                              SrcDvcIpAddr = tostring(Data.IpAddress),
+                                              TargetPortNumber = toint(Data.IpPort),
+                                              LogonGuid = tostring(Data.LogonGuid),
+                                              LogonType = tostring(Data.LogonType),
+                                              ActingProcessCreationTime = Data.ProcessCreationTime,
+                                              ActingProcessId = tostring(toint(Data.ProcessId)),
+                                              ActingProcessName = tostring(Data.ProcessName),
+                                              Status = tostring(Data.Status),
+                                              ActorUserDomainName = Data.SubjectDomainName,
+                                              ActorSessionId = tostring(Data.SubjectLogonId),
+                                              ActorUsername = tostring(Data.SubjectUserName),
+                                              ActorUserId = tostring(Data.SubjectUserSid),
+                                              SubStatus = tostring(Data.SubStatus),
+                                              TargetDomainName = tostring(Data.TargetDomainName),
+                                              TargetSessionId = tostring(Data.TargetLogonId),
+                                              TargetUserId = tostring(Data.TargetUserSid),
+                                              TargetUsername = tostring(Data.TargetUserName),
+                                              SrcDvcHostname = tostring(Data.WorkstationName),
+                                              ActorUserAccount = tostring(Data.SubjectAccount),
+                                              TargetUserAccount = tostring(Data.TargetAccount),
+                                              EventProduct = "Microsoft-Windows-Security-Auditing"
+                                  | extend EventStatus= iff(SubStatus=='0x0',Status,SubStatus)
+                                  // -- creating EventMessage matching EventMessage in SecurityEvent table
+                                  | extend EventMessage = case(EventID == 4634, "4634 - An account was logged off.", 
+                                                              EventID == 4625, "4625 - An account failed to log on.",
+                                                              EventID == 4624 ,"4624 - An account was successfully logged on.",
+                                                              "4647 - User initiated logoff."),
+                                          EventResult = iff(EventID == 4625, 'Failure', 'Success')
+                                  | project-rename 
+                                      TargetDvcHostname = Computer
+                                  // , TargetUserType=AccountType - no AccountType in windowsEvents
```

Comment:
Did you follow with engineering on those issues?
- I recall UID was a bug.
- AccountType: what is the enrichment done in SecurityEvents?

Reply:
talked about it - will be added later to the original schema